src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -161,101 +156,102 @@ public void onDye(PlayerInteractEntityEvent e) {
     }
 
     @EventHandler
-    public void onMiningDrillUse(PlayerInteractEvent e) {
-        final Block block = e.getClickedBlock();
+    public void onDrillUse(PlayerInteractEvent event) {
+        final Block block = event.getClickedBlock();
         if (block == null) {
             return;
         }
 
         final Material blockType = block.getType();
         final Location blockLocation = block.getLocation();
+        final ItemStack hand = event.getItem();
 
         final MiningDrill miningDrill = (MiningDrill) SlimefunItem.getById(Items.MINING_DRILL.getItemId());
+        if (miningDrill.isItem(hand)) {
+            Validate.notNull(miningDrill);
```

Comment:
This validate check won't fire, if `miningDrill` is null, then the if-statement will already trigger a NPE.
Either place the validate check before this, use `Objects.notNull` inside the if-statement or remove the validate check.

Reply:
Thanks for the feedback <3

Reply:
Thanks for the feedback